galvani/BioLogic.py

```diff
@@ -27,9 +27,13 @@ def fieldname_to_dtype(fieldname):
                        "counter inc."):
         return (fieldname, np.bool_)
     elif fieldname in ("time/s", "P/W", "(Q-Qo)/mA.h", "x", "control/V",
-                       "control/V/mA", "(Q-Qo)/C", "dQ/C", "freq/Hz",
-                       "|Ewe|/V", "|I|/A", "Phase(Z)/deg", "|Z|/Ohm",
-                       "Re(Z)/Ohm", "-Im(Z)/Ohm"):
+                       "control/mA", "control/V/mA", "(Q-Qo)/C", "dQ/C",
```

Comment:
This looks messy (which is my fault). I created a separate issue to refactor it: https://github.com/echemdata/galvani/issues/72